Documentation/guides/OneDotNetSingleProject.md

```diff
@@ -29,6 +29,11 @@ Xamarin.Android and Xamarin.iOS/Mac SDKs:
 * `$(ApplicationTitle)` maps to `/application/@android:title` or
   `CFBundleDisplayName`
 
+Added in Xamarin.Android 12.2 / .NET 7:
```

Comment:
As per another comment, ix-nay on the "Xamarin.Android"-ay.  And it would be 13.2, not 12.2.  Were it to exist.